### PR TITLE
Delete removed contacts

### PR DESCRIPTION
**Pull request by annando**
We hadn't handled it very well when a user on a remote site decided to delete their profile. We not only detected it in the most cases, but we hadn't even detected it when probing the contact - and even if that had worked, the deletion most likely would had failed.

This fixes this all.

### CODE REVIEW

File: src/Protocol/ActivityPub.php [hunk 1]
```diff
@@ -113,7 +113,10 @@ private static function getAccountType($apcontact)
 			case 'Application':
 				$accounttype = User::ACCOUNT_TYPE_RELAY;
 				break;
-		}
+			case 'Tombstone':
+				$accounttype = User::ACCOUNT_TYPE_DELETED;
+				break;
+			}
```
Comment (MrPetovan) [inline src/Protocol/ActivityPub.php]: Please restore the original indentation.
```suggestion
		}
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Protocol/ActivityPub.php (hunk 1 above)
Comment (annando) [reply src/Protocol/ActivityPub.php]: Done

**Review by MrPetovan — APPROVED** [review]

**Comment by AlfredSK** [comment]
After git pulling this I'm getting a lot of that...

[PHP warnings](https://github.com/friendica/friendica/issues/9251#issuecomment-746715944)

**Comment by annando** [comment]
@AlfredSK this should not really be related since we only trigger the deletion here.

**Comment by AlfredSK** [comment]
...and we do not try to fetch the public key of the contact during/for the deletion?

I mean it would be a strange coincidence. No such warnings, git pull, hundreds of such warnings. Hmm.

**Comment by AlfredSK** [comment]
I added the corresponding error from Friendica log in the PHP warnings issue.

**Comment by annando** [comment]
This happens when we are probing a contact.

**Comment by AlfredSK** [comment]
Ok. I was concerned because it happens on all my 4 nodes apparently everytime cron starts the worker.

**Pull request merged**